internal/cmd/root.go

```diff
@@ -60,8 +60,8 @@ func initConfig() {
 	}
 
 	viper.AddConfigPath(configFolder)
-	viper.SetConfigType(config.DEFAULT_CONFIG_FILE_TYPE)
-	viper.SetConfigName(config.DEFAULT_CONFIG_FILE_NAME)
+	viper.SetConfigType(config.DefaultConfigFileType)
```

Comment:
Go uses CamelCase for identifiers not snake_case. See https://go.dev/wiki/CodeReviewComments#mixed-caps